sig-contributor-experience/devstats/OWNERS

```diff
@@ -1,13 +1,12 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
-reviewers:
-  - dims
-  - nikhita
 approvers:
-  - lukaszgryglicki
+  - dims
   - jberkus
   - LappleApple
+  - mrbobbytables
 emeritus_approvers:
+  - lukaszgryglicki
```

Comment:
@lukaszgryglicki 

Would you like to remain as an emeritus approver in the devstats owners file?